chapter_13_database_layer_validation.asciidoc

```diff
@@ -784,7 +831,17 @@ OK
 ----
 
 
-Let's try a full FT run:
+Let's try a full FT run: they're all passing! We can remove the early return
+now.
```

Comment:
shall we add a command output showing the fts passing, before we remove the early return?

Reply:
I think just telling the reader they are passing should be enough, especially because running them again to find the previous issues in the test is pretty much the same step. Also, the 
```
Ran 4 tests in 20 s

OK
```
output is pretty straightforward. I can add it if you want, but I think it isn't necessary.

Reply:
it's not so much for the readers, more for the book's tests - this guarantees that we have passing code/tests at this point in the book.

Reply:
in that case, I'll add it of course